src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -128,3 +129,26 @@ export function submit(form, formConfig) {
     return Promise.reject(respOrError);
   });
 }
+
+// TODO: Get the actual event names from Nedie
+const applicantRelationshipEventMap = {
+  Myself: 'myself',
+  'Someone else': 'someone-else',
+  Anonymous: 'anonymous',
+};
+
+// This is a hack. This function is intended to be used as a uiSchema's
+// ui:description callback. We're simply taking advantage of the fact that
+// ui:description callbacks are fired whenever the form data changes so that
+// we can check the current state of the form data and register what option the
+// user selected for who they are filing feedback on behalf of.
+export function recordApplicantRelationship({ formData }) {
+  const onBehalfOf = _.get('onBehalfOf', formData);
+  if (applicantRelationshipEventMap[onBehalfOf]) {
```

Comment:
We will likely want to use a react lifecycle method such as [`componentDidUpdate`](https://reactjs.org/docs/react-component.html#componentdidupdate), to ensure that this event only fires when a change is made (by comparing prevProps with current props), and not every time an applicant relationship field value exists.

Reply:
That totally makes sense! Thanks.